frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -170,6 +173,35 @@ export function PluginDrawer(): JSX.Element {
                                 </div>
                             ) : null}
 
+                            {user?.organization?.plugins_access_level === PluginsAccessLevel.Root &&
+                                user?.is_multi_tenancy && (
```

Comment:
Q: Will this mean that this is a cloud user or...?

Also: Extract function `canEnableGlobalPlugins`.

Reply:
Didn't extract the `is_multi_tenancy` since it'd require passing in `user` too and it's not globally relevant to permissioning. Just a presentation thing for now. Did extract the Root (etc.) check though.